components/measurement/DetailsBox.js

```diff
@@ -63,11 +63,11 @@ export const DetailsBox = ({ title, content, collapsed = false, ...rest }) => {
   return (
     <StyledDetailsBox width={1} {...rest} mb={3}>
       {title &&
-        <StyledDetailsBoxHeader px={3} py={2} bg='gray2' alignItems='center' onClick={onToggle}>
+        <StyledDetailsBoxHeader px={3} py={2} bg='gray2' alignItems='center' >
           <Box>
             <Heading h={4}>{title}</Heading>
           </Box>
-          <Box ml='auto'>
+          <Box ml='auto' onClick={onToggle}>
             <CollapseTrigger size={36} isOpen={isOpen} />
```

Comment:
Looks like the reason the CollapseTrigger icon doesn't spin anymore is because this prop name was changed. Can you rename this to `open={isOpen}`? That should fix it.